src/content/en/fundamentals/accessibility/00-intro/intro-three.md

```diff
@@ -0,0 +1,91 @@
+
+ - *Great, let's talk about hearing impairments.*
+
+>Next we have hearing impaired users. This group can range from the profoundly deaf through to hard-of-hearing. And much like eyesight, our hearing tends to degrade with age.
+>
+>For hearing-impaired users we need to make sure that we're not relying on sound, so making sure to use things like video captions and transcripts, and providing some kind of alternative, if sound is part of the interface.
+>
+>And like we saw with vision and motor impairments, it's really easy to imagine a situation where someone whose ears work fine would benefit from these accommodations as well. A lot of my friends say they love it when videos have captions and transcripts because it means that if they're in an open plan office and didn't bring their headphones, they can still watch the video!
+
+ - *All right, can you tell us a bit about cognitive impairments?*
+
+>Finally there's a range of cognitive conditions like ADD, Dyslexia, and Autism, which can mean that people want or need to access things differently. The accommodations for these groups are obviously extremely diverse, but we definitely find some overlap with other areas, like using zoom functionality to make reading or concentrating easier. Also, these users may find that really minimal design works best because it minimizes distraction and cognitive load. 
+>
+>I think everyone can relate to the stress of cognitive overload, so it's obvious that if we create something that works well for someone with a cognitive impairment, we're going to be creating something which is going to be a pleasant experience for everyone.
+
+ - *So, how would you summarize how you think about accessibility?*
+
+>When you look at the broad range of abilities and disabilities that people might have, you can see that designing and building products only for people who have perfect vision, hearing, dexterity, and cognition seems incredibly narrow. It's almost self-defeating, because we're creating a more stressful and less usable experience for everyone, and for some users creating an experience which actually excludes them altogether.
+
+In this interview, Victor identified a range of impairments, and placed them into four broad categories: *visual*, *motor*, *hearing*, and *cognitive*. He also pointed out that each type of impairment might be *situational*, *temporary*, or *permanent*. 
+
+Let's take a look at some real-world examples of access impairments and see where they fall within those categories and types. Note that some impairments may fall into more than one category or type.
+
+| &nbsp;        | Situational  | Temporary | Permanent |
```

Comment:
The formatting on this table is very hard to read. @petele are there better built-in table styles for the docs?